spython/instance/cmd/stop.py

```diff
@@ -8,13 +8,15 @@
 from spython.logger import bot
 
 
-def stop(self, name=None, sudo=False):
+def stop(self, name=None, sudo=False, timeout=10):
```

Comment:
E.g., here we would set `timeout=None` and update the docstring to say default uses Singularity default (unset), and then for the if just do:

```python
if "version 3" in self.version() and timeout:
    subgroup = ["instance", "stop", "-t", str(timeout)]
```

But I'm curious and want to open this up for discussion - have you found in practice that the default singularity timeout is not sufficient for most cases? If this is the case, then we likely would help users by having the default timeout of 10 (and the current implementation is good). Let me know your thoughts!

Reply:
The timeout is very useful when running applications that are making a lot of I/O at stop time: a common use case can be an in-memory system (plenty of GBs in memory) that need to offload data at once a computation is finished (plenty of GBs to write) requiring in most cases more than 10 seconds